include/vzlist.h

```diff
@@ -117,7 +119,8 @@ struct Cveinfo {
 	unsigned long long features_mask;
 	unsigned long long features_known;
 	float vm_overcommit;
-	unsigned long nf_mask; /* netfilter */
```

Comment:
This style from https://github.com/kolyshkin/vzctl/blob/master/include/res.h#L36
but yes- in vzlist.h comments look like
```
int layout; // VE_LAYOUT_*
```
I will change the comment style.